include/af/util.h

```diff
@@ -128,7 +128,7 @@ namespace af
 #endif
 
     // Purpose of Addition: "How to add Function" documentation
-    AFAPI array exampleFunction(const array& in, const af_someenum_t param);
+    AFAPI array exampleFunction(const array& a, const array& b, const af_someenum_t param);
```

Comment:
This function doesn't serve any other purpose than tutorial help. Shall i revert it back and just use single input array to do `a+a` style operation ?